core/src/main/java/io/grpc/internal/GrpcAttributes.java

```diff
@@ -50,6 +50,13 @@
   public static final Attributes.Key<Boolean> ATTR_LB_PROVIDED_BACKEND =
       Attributes.Key.create("io.grpc.grpclb.lbProvidedBackend");
 
+  /**
+   * The authority to be used when constructing {@link io.grpc.Channel}s to this this EquivalentAddressGroup.
```

Comment:
this this -> this

Also did you mean to say Subchannel instead of Channel (which makes sense given the main PR comment.

Reply:
I went for Channel as the Subchannel could not be referenced from this context. I changed this to Subchannel and removed the javadoc link.